sigh/lib/sigh/download_all.rb

```diff
@@ -19,7 +19,12 @@ def download_all
 
     def download_profile(profile)
       FileUtils.mkdir_p(Sigh.config[:output_path])
-      profile_name = "#{profile.class.pretty_type}_#{profile.app.bundle_id}.mobileprovision" # default name
+
+      if Sigh.config[:use_apple_developer_portal_profile_names]
```

Comment:
This change has nothing to do with the pagination support, right? If so, can you move that out into a separate PR?